### ISSUE TITLE
Add jekyll blog

### ISSUE THREAD

**Issue by ultrasaurus** [bridgefoundry/bridgefoundry.github.io#7]
I was thinking we should start with a blog tab, then create a different issue to have latest posts featured on the home page or something

sent separate note via email to @tsykoduk -- my proposal is to not use blog.bridgefoundry.org moving forward (simply reverting that content to RailsBridge, since we haven't written any blog posts since switching the name, it appears to have no inbound links:  google search for link:blog.bridgefoundry.org yields no search results, so I think it is safe not to redirect from there

**Comment by ultrasaurus** [comment]
@rachelmyers want to get started on this solo or find a time to pair together?

**Comment by tsykoduk** [comment]
@ultrasaurus Sounds good to me. Do we want to use the new host moving forward? If so, what's the URL/IP there?

**Comment by rachelmyers** [comment]
I can pull the basic blog together on my own. (I'm a bit worried about time over the next few weeks. I can definitely create the blog, but I won't be able to dote on it until February.) :hourglass_flowing_sand:

**Comment by ultrasaurus** [comment]
@tsykoduk you could just swap back to blog.railsbridge.org for now, then @lilliealbert can let you know when they get a new host for RailsBridge

**Comment by ultrasaurus** [comment]
@rachelmyers why don't I draft some content in parallel, then there'll be something for you to post!  Sometime in the next few weeks or early feb is fine for Bridge Foundry, just didn't want to hold up the RailsBridge folks who are excited about blogging on the separate domain

**Comment by ultrasaurus** [comment]
FYI:  @rachelmyers  won't have time to devote to this till March, so would love it if anyone wants to dive in and help!

**Comment by Vibhuti** [comment]
I am working on it!

**Comment by ultrasaurus** [comment]
@Vibhuti wonderful. I'll get to work on the content for the first post!

**Comment by Vibhuti** [comment]
@ultrasaurus basic blog functionality is working, but somehow bootstrap is not working with the blog post. Trying to figure out.

**Comment by Vibhuti** [comment]
I added blog tab in the nav bar which will show all blog's  title with their corresponding date. After clicking the blog title it will show blog post.

**Comment by Vibhuti** [comment]
@ultrasaurus bootstrap is working with blog post also, as i added all bootstrap files to public folder. I think we need to improve it. https://github.com/bridgefoundry/bridgefoundry.github.io/pull/9

**Comment by Vibhuti** [comment]
Fixed the issues! https://github.com/bridgefoundry/bridgefoundry.github.io/pull/9

**Comment by ultrasaurus** [comment]
merged into this branch:  https://github.com/bridgefoundry/bridgefoundry.github.io/tree/blog
